webpack.common.js

```diff
@@ -29,7 +29,10 @@ module.exports = {
     resolve: {
         modules: [
             path.resolve(__dirname, 'node_modules')
-        ]
+        ],
+        alias: {
+            'SyncPlay': path.resolve(__dirname, 'src/components/syncPlay/core')
```

Comment:
This is probably not a good practice now that we have proper module support.

Reply:
Indeed, please avoid aliasing modules after we spent month un-aliasing the entire source :smile:

Reply:
Ok, it does look like I'm missing the old Emby way but I swear I'm not hahaha. I've split the code into core and ui so that I could reuse core in the Chromecast client as well; I was thinking about making another repo so I don't have to copy files around. 

The alias is a sort of placeholder for that package. But this won't happen for this PR so I guess having it there isn't that useful.

Reply:
If you could remove the alias, that would be great! I will approve once that change is made.